packages/monaco/src/browser/monaco-text-model-service.ts

```diff
@@ -127,12 +127,15 @@ export class MonacoTextModelService implements ITextModelService {
 
     protected readonly modelOptions: { [name: string]: (keyof ITextModelUpdateOptions | undefined) } = {
         'editor.tabSize': 'tabSize',
-        'editor.insertSpaces': 'insertSpaces'
+        'editor.insertSpaces': 'insertSpaces',
+        'editor.indentSize': 'tabSize'
```

Comment:
Question: Is it correct that both set the `tabSize` property of the monaco text model?

Reply:
Good catch! I switched indeed to 'indentSize'.